### PR TITLE
add tracking to cells on Home page and search result text tweaks

### PR DESCRIPTION
**Pull request by emarc0314**
## Overview

Add tracking to cells on Home Page, made changes to "Results" label constraints on search page

## Changes Made

In HomeTableViewCell a popularityImageView and popularityLabel were added to show how many people were tracking a given course on a cell in the Home Page. Also, when searching the "Results" text was indented too far down, so I changed the constraints so that it was closer to the search bar.

## Test Coverage

Ran the app on my phone and asked a friend if it looked good

## Screenshots (delete if not applicable)

<details>

  <summary>Results</summary>

IMG_0162  
</details>

<details>
  <summary>Cell Updates</summary>

IMG_0163

</details>

### CODE REVIEW

File: CourseGrab/ViewControllers/Home/HomeTableViewCell.swift [hunk 1]
```diff
@@ -131,6 +140,17 @@ class HomeTableViewCell: UITableViewCell {
             make.top.leading.equalTo(18)
             make.trailing.equalToSuperview().inset(40)
         }
+        
+        popularityImageView.snp.makeConstraints { make in
+            make.top.equalTo(courseLabel.snp.bottom).offset(9)
+            make.trailing.equalTo(popularityLabel.snp.leading).offset(-6)
```
Comment (danielVebman) [inline CourseGrab/ViewControllers/Home/HomeTableViewCell.swift:146]:
```suggestion
            make.trailing.equalTo(popularityLabel.snp.leading).inset(6)
```
Does this work?

**Review by danielVebman — APPROVED** [review]
Clean 🔔!

File: CourseGrab/ViewControllers/Home/HomeTableViewCell.swift [hunk 2]
```diff
@@ -145,6 +165,8 @@ class HomeTableViewCell: UITableViewCell {
         sectionLabel.text = section.getSectionByTimezone()
         statusBadge.image = section.status.icon
         titleLabel.text = "\(section.subjectCode) \(section.courseNum): \(section.title)"
+        popularityLabel.text = "\(section.numTracking) tracking"
+
```
Comment (Npikielny) [inline CourseGrab/ViewControllers/Home/HomeTableViewCell.swift:169]: Delete new line

**Review by Npikielny — APPROVED** [review]
Wonderful, fantastic, lovely, amazing, fantabulous, inspiring 🤩

**Review by haiyingweng — APPROVED** [review]
looks great 🚀

**Review by spd75 — COMMENTED** [review]
Other comments say what I was thinking

**Pull request merged**